js/modules/k6/k6.go

```diff
@@ -36,6 +36,12 @@ import (
 
 type K6 struct{}
 
+// ErrGroupInInitContext is returned when group are using in init context
+var ErrGroupInInitContext = errors.New("Using group in init context is not supported")
```

Comment:
Missing `the` - `in the init context` sounds better. Also, `group` should probably be `group()`.